### PR TITLE
Update WebSocket sender to avoid extra utf-8 decode/encode round-trip

### LINKED ISSUE #676: Switch `send_json` to use `send_frame` once aiohttp 3.11 is available to avoid extra decode/code cycle
Currently we [decode the json bytes](https://github.com/home-assistant-libs/aioshelly/blob/da3b1c9cf496d137968fa432da4247848ecc33fc/aioshelly/json.py#L24), than aiohttp re-encodes them back to bytes before sending. We can avoid this in aiohttp 3.11 by using `send_frame(bytes, WSMsgType.TEXT)` via https://github.com/aio-libs/aiohttp/pull/9348

https://github.com/home-assistant-libs/aioshelly/blob/da3b1c9cf496d137968fa432da4247848ecc33fc/aioshelly/rpc_device/wsrpc.py#L534C37-L534C47

### PR DESCRIPTION
**Pull request by bdraco**
closes #676

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 bluetooth-data-tools>=1.19.0
-aiohttp
+aiohttp>=3.11.0
```
Comment (bdraco) [inline requirements.txt:2]:
```suggestion
aiohttp>=3.11.1
```

**Review by bieniu — APPROVED** [review]
Looks good, tested with real device.

**Review by chemelli74 — APPROVED** [review]
Quick tests shows green light

**Comment by bdraco** [comment]
No rush to release this. It can come in whenever we do another bump.

**Comment by bdraco** [comment]
Thanks for the 👀

**Pull request merged**